app/models/setting.rb

```diff
@@ -0,0 +1,16 @@
+# RailsSettings Model
```

Comment:
Can you add a little bit more of a description to this model please? Would love for us to get in the habit of nice top-level comments (without going OTT) with future devs (us) in mind :)